### ISSUE TITLE
Won't sync in VSCode 1.23

### ISSUE THREAD

**Issue by duartealexf** [shanalikhan/code-settings-sync#544]
**Visual Studio Code Version : 1.23
**Code Settings Sync Version : 2.9.2
**Operating System : MacOS 10.13.4
**Occurs On: Upload
**Proxy Enabled: No
**GIst Id: 247e80e049a5e76b831536e69005a353

Updated to VSCode 1.23 and this error started happening:

![image](https://user-images.githubusercontent.com/5911016/39662330-408c17d8-5036-11e8-9ddc-7111671ca4b4.png)

And in settings, all of the sync.* config are marked as unknown:

![image](https://user-images.githubusercontent.com/5911016/39662340-70525428-5036-11e8-84a1-cb0dad8a3fc9.png)

**Comment by jasonekratz** [comment]
Just as a counterpoint to the folks having problems since 1.23 I am not having the same problems with my copy of 1.23 and Sync 2.9.2. Not saying they're not having problems....just that I'm not ;)

**Comment by shanalikhan** [comment]
Try removing extension, restarting code and then install extension

**Comment by duartealexf** [comment]
I did that, still not working.

**Comment by duartealexf** [comment]
I think it's some problem related to VSCode, because a lot of extensions stopped working, even some native ones like parsers and git integration.

I fixed it by:

- Completely reinstalling VSCode (https://medium.com/@jimkang/complete-uninstall-remove-vscode-mac-5e48bef3bdec)
- Reinstalled my extensions one by one (I saw which ones were installed by looking at the old settings from the gist).
- Copied my settings.json from the gist and pasted into new installation's settings.json.
- Reconfigured Settings Sync extension (created a new token and re-uploaded).

So far so good.
